### ISSUE TITLE
Gathering metrics from Liberty runtime with Prometheus (MP19)

### ISSUE THREAD

**Issue by lauracowen** [OpenLiberty/docs#334]
Based on https://openliberty.io/blog/2018/10/05/microprofile-faulttolerance-1.1-metrics.html (but link to #273 reference topic for a catalog of possible metrics instead of listing them again here). Don't talk about versions of MP or OL though. Just assume the latest version.

Target audience is an ops person (not developer) who is trying to work with the metrics emitted by a microservice in production.

The blog post above is specifically about fault tolerance metrics but see if you can make it relevant to any of the other metrics too (even if it's just by being a bit more generic then using the FT metrics scenario as the example).

When done, we'll redirect Andy's blog post to this topic.

See the blog post for a list of relevant guides, plus the metrics guide: https://openliberty.io/guides/microprofile-metrics.html

**Comment by Charlotte-Holt** [comment]
@lauracowen I went ahead and got these steps (with some editing, etc.) into AsciiDoc (https://draft-openlibertyio.mybluemix.net/docs/ref/general/#gathering_displaying_metrics.html). Based on what we'd talked about, I think it was okay for me to use these FT examples for the purposes of this topic. I think I probably need to add a little more meat at the top to make sure that it's clear that ops people can apply the steps from the examples to any types of metric data (not just FT). Needs a closing sentence too, I think. Could you take a look and give me your initial thoughts, and I'll update this one again today too? Thanks!!

**Comment by Charlotte-Holt** [comment]
I worked on this for 19.0.0.7, but we decided that we need to prioritize dev-focused topics, and this topic is more ops-focused, so I'm moving it back to the backlog to complete at a later time. 

The most recent version is built to the draft site at https://draft-openlibertyio.mybluemix.net/docs/ref/general/#gathering-displaying-metrics.html. It was adapted and generalized from @Azquelt's blog post, but more work still needs to be done to include relevant and coherent examples of creating graphs in Grafana.
